q2_moshpit/plugin_setup.py

```diff
@@ -1122,3 +1185,10 @@
                 "classification of NGS reads.",
     citations=[citations["menzel2016"]],
 )
+
+plugin.register_semantic_types(BUSCOResults)
+plugin.register_semantic_type_to_format(
+    BUSCOResults,
+    artifact_format=BUSCOResultsDirectoryFormat)
+plugin.register_formats(BUSCOResultsFormat, BUSCOResultsDirectoryFormat)
+importlib.import_module('q2_moshpit.busco.types._transformer')
```

Comment:
Just because this is the first time we define semantic types and formats in moshpit I will ask this... do we want to keep this here or should we move it to q2-types? I anyways was working on a semantic type for the busco database in q2-types (i think that PR is pending).

Reply:
Good point - I figured I'll put it here as I really don't see a use case when you'd need the BUSCOResults type without q2-moshpit installed. Following this reasoning, though, I'm wondering now where the database type should really live 🤦

Reply:
I see. Well, it definitely would speed up our development process to have the database semantic type here no? Since we are more in control of the PRs here than in q2-types? I gues that would be a point in favor of having it here.

Reply:
Yeah, sounds very sensible - let's keep the BUSCO-related one here then 👍